commons/connector/upload/altoida/altoida_summary.avsc

```diff
@@ -13,9 +13,17 @@
           "name": "GenderType",
           "type": "enum",
           "doc": "Gender types provided by Altoida.",
-          "symbols": ["MALE", "FEMALE", "OTHER"]}, "doc": "Gender of the participant.", "default": "OTHER" },
-    {"name": "classification", "type": "int", "doc": "Classification based on the analysis  0=healthy, 1=at risk, 2=MCI due to AD."},
+          "symbols": ["MALE", "FEMALE", "OTHER"]}, "doc": "Gender of the participant.", "default": "OTHER"},
```

Comment:
Here add null or UNKNOWN default to distinguish ambiguous gender and unspecified gender.